examples/ethernet/ethernet.ino

```diff
@@ -0,0 +1,49 @@
+#include <simpleRPC.h>
+#include <Ethernet.h>
+
+byte mac[] = { 0xDE, 0xAD, 0xBE, 0xEF, 0xFE, 0xED };
+IPAddress ip(192, 168, 1, 50);
+EthernetServer server(10000);
+
+byte ping(byte data) {
+  return data;
+}
+
+void connectEthernet() {
+  if (Ethernet.hardwareStatus() == EthernetNoHardware) {
+    while (true) {
+      delay(1);  // do nothing, no point running without Ethernet hardware
+    }
```

Comment:
What about this one:

    while (Ethernet.hardwareStatus() == EthernetNoHardware) {
      delay(1);
    }